### DISCUSSION TITLE
What's the best way to get ComfyUI install manager working?

### DISCUSSION THREAD

**Discussion by ataylorm** [Stability-AI/StableSwarmUI#136] [Q&A] [answered]
I have found that installing nodes for ComfyUI is a PITA since it doesn't seem to work automatically when using through StableSwarmUI.  Instead I have to put them into ComfyUI portable, start it up so it "builds them" or what ever it's doing, and then copy the folders manually to the ComfyUI folder on the backend of StableSwarm.

**Comment by mcmonkey4eva** [comment, marked as answer]
You mean https://github.com/ltdrdata/ComfyUI-Manager ? You should be able to just install that into swarm's comfy (git clone it into `Swarm/dlbackends/comfy/ComfyUI/custom_nodes`) and use it within the comfy tab, it should just work.
